.github/workflows/build-windows.yml

```diff
@@ -127,21 +127,23 @@ jobs:
           import os
 
           workspace = os.environ["GITHUB_WORKSPACE"]
-          triplets = os.path.join(workspace, "external", "vcpkg", "triplets")
-
-          for path, dnames, fnames in os.walk(triplets):
-            for fname in fnames:
-              fname = os.path.join(path, fname)
-              print(fname)
-              with open(fname, "rb") as handle:
-                contents = handle.read()
-              # If a file is already patched we skip patching it again because
-              # this affects the hashes calculated by vcpkg for caching
-              if b"VCPKG_BUILD_TYPE release" in contents:
-                continue
-              contents += b"\nset(VCPKG_BUILD_TYPE release)\n"
-              with open(fname, "wb") as handle:
-                handle.write(contents)
+          triplet_paths = [os.path.join(workspace, "external", "vcpkg", "triplets"),
```

Comment:
As follow-up I would _really_ like to
- get this script moved to a .py file
- run uniformly from both workflows (`python set-vcpkg-release-tripley.py $WORKSPACE/...`)
- remove the `patch_...` script below

Reply:
Done in #4439.